conversion/conversion.go

```diff
@@ -73,10 +73,10 @@ var (
 	MaxWorkers = 20
 )
 
-func SchemaConv(driver string, targetDb string, ioHelper *IOStreams, schemaSampleSize int64) (*internal.Conv, error) {
+func SchemaConv(driver, sqlConnectionStr, targetDb string, ioHelper *IOStreams, schemaSampleSize int64) (*internal.Conv, error) {
```

Comment:
These public functions now deserve a comment - even if it just explains the behavior for sqlConnectionStr (which is that it's expected to be empty for dump files and dynamodb -- and for direct connect to mysql and postgres, empty value reflects that the command is being executed from legacy mode with global flags). Similarly for DataConv.